components/resources/sounds.js

```diff
@@ -0,0 +1,4 @@
+const dropSound = document.getElementById("sound-drop");
+const shiftSound = document.getElementById("sound-shift");
+const clearLineSound = document.getElementById("sound-clear_line");
+const gameOverSound = document.getElementById("sound-game_over");
```

Comment:
How about this?
```javascript
const _dropSound = document.getElementById("sound-drop");
const _shiftSound = document.getElementById("sound-shift");
const _clearLineSound = document.getElementById("sound-clear_line");
const _gameOverSound = document.getElementById("sound-game_over");

class SoundResources {
    static dropSound() {
        return _dropSound
    }
    static shiftSound() {
        return _shiftSound
    }
    static clearLineSound() {
        return _clearLineSound
    }
    static gameOverSound() {
        return _gameOverSound
    }
}
```
or
```javascript
class SoundResources {
    static dropSound = document.getElementById("sound-drop");
    static shiftSound = document.getElementById("sound-shift");
    static clearLineSound = document.getElementById("sound-clear_line");
    static gameOverSound = document.getElementById("sound-game_over");
}
```

to clarify where the resource is located or improve readability.

But I think this is not necessary.